datascience-notebook/Rprofile.site

```diff
@@ -0,0 +1,5 @@
+
+# Add mimetype to specifies how the plot returns from R to the browser.
+# https://notebook.community/andrie/jupyter-notebook-samples/Changing%20R%20plot%20options%20in%20Jupyter
+
+options(jupyter.plot_mimetypes = c('text/plain', 'image/svg+xml', 'application/pdf'))
```

Comment:
According to @benz0li there is no need for png and jpeg, the docs state that it's a default but the [tests](https://github.com/IRkernel/IRkernel/blob/a862774dc0758b23ef4e911a3c0b1d9e550652bc/tests/testthat/test-options.r) suggest that  `text/plain` and `image/png` are the defaults. So it's very confusing ATM.

Reply:
I will check tonight and get back to you.

Yes, the default settings are confusing; both for `IRKernel` and `IPython` – the latter stating the default being `retina`.

Default seems to be `image/png` for both plus `text/plain`, which is a non-removable default.

Reply:
Writing this, I wonder why `inkscape` was added at all...

Reply:
Manually doing `apt-get install inkscape` fixes the problem. We can throw it into minimal-notebook where pandoc and the other nbconvert prereqs are.  Or we can put this only in images that use SVGs by default like r- and datascience-notebook.

I think the former is easier to reason about even though it bloats the minimal-notebook image by another ~30 MB.

_Originally posted by @parente in https://github.com/jupyter/docker-stacks/issues/243#issuecomment-234160567_

Reply:
> Or we can put this only in images that use SVGs by default like r- and datascience-notebook.

Do the r- and datascience-notebook truly use SVGs by default? I don't see any code modifying _figure formats_ or _plot mimetypes_.

Reply:
I'm quite sure `inkscape` can be safely removed, if `pdf` is added to _figure formats_ and `application/pdf` to _plot mimetypes_.
ℹ️ `nbconvert` will use PDF when converting to PDF.

`image/svg+xml` is a nice to have for plot display in the browser.
ℹ️ `nbconvert` would use SVG instead of PNG when converting to HTML.

P.S.: See https://github.com/JuliaPlots/Makie.jl/issues/2195#issuecomment-1210494177 for the discussion about vector graphics becoming very large and slow for plots with lots of data points.